src/jit/gentree.cpp

```diff
@@ -13832,6 +13832,16 @@ regNumber ReturnTypeDesc::GetABIReturnReg(unsigned idx)
             }
         }
     }
+
+#elif defined(_TARGET_X86_) && !defined(LEGACY_BACKEND)
```

Comment:
Here we don't need !defined(LEGACY_BACKEND), since this routine is correct even for legacy backend.